packages/lisk-constants/test/index.ts

```diff
@@ -22,7 +23,7 @@ import {
 	TESTNET_NETHASH,
 	MAINNET_NETHASH,
 	SIGNED_MESSAGE_PREFIX,
-} from '../src';
+} from '../src/index';
```

Comment:
I prefer to keep index out. Is there a reason that you added it ?

Reply:
it seems in typescript, u need to specify index

Reply:
It shouldn't be like that. http://www.typescriptlang.org/docs/handbook/module-resolution.html  (How TypeScript resolves modules section). I don't get any linter or build errors when I remove it.